### PR TITLE
Implement cd command

### LINKED ISSUE #135: Implement a cd Command to manipulate current path in model

### PR DESCRIPTION
**Pull request by Ma-Yueran**
Currently the not many tests are added for the command and its parser.

Themes of UI are updated to fit file explorer

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/CdCommand.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+    private AddressType addressType;
+    private String addressString;
+
+    /**
+     * Creates a CdCommand to change the current path of the file explorer to the given {@code FileAddress}.
+     */
+    public CdCommand(AddressType addressType, String addressString) {
+        requireNonNull(addressType);
+        requireNonNull(addressString);
```
Comment (HynRidge) [inline src/main/java/seedu/address/logic/commands/CdCommand.java]: I think requireNonNull(addressType) and requireNonNull(addressString) can be combined into requireNonNull(adddressType, addressString);

**Review by HynRidge — CHANGES_REQUESTED** [review]
Nice implementation, and what a neat code..
There is just some minor changes needed!!
LGTM..

File: src/main/java/seedu/address/logic/commands/CdCommand.java (hunk 1 above)
Comment (Ma-Yueran) [reply src/main/java/seedu/address/logic/commands/CdCommand.java]: Ok, I will change that, thank you for reviewing the code!

File: src/test/java/seedu/address/model/explorer/FileListStub.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package seedu.address.model.explorer;
+
+public class FileListStub extends FileList {
```
Comment (anthony6401) [inline src/test/java/seedu/address/model/explorer/FileListStub.java]: What is this FileListStub for? I can't find any class that use this. If this class is not being used, can it be deleted? We can add it again later if we really need it, to make the code cleaner.

**Review by anthony6401 — CHANGES_REQUESTED** [review]
Great job at implementing the CdCommand!
There is some minor things to be addressed.

File: src/test/java/seedu/address/model/explorer/FileListStub.java (hunk 2 above)
Comment (Ma-Yueran) [reply src/test/java/seedu/address/model/explorer/FileListStub.java]: Oh, yeah, forgot to delete that

**Review by anthony6401 — APPROVED** [review]
LGTM!

**Review by HynRidge — APPROVED** [review]
LGTM

**Pull request merged**